config/crd/bases/rbac-operator.argoproj-labs.io_argocdroles.yaml

```diff
@@ -77,8 +81,10 @@ spec:
             description: ArgoCDRoleStatus defines the observed state of Role
             properties:
               argocdRoleBindingRef:
+                description: argocdRoleBindingRef defines the reference to the ArgoCDRoleBinding Resource.
```

Comment:
add to api/v1alpha1/argocdrole_types.go please :)

Reply:
done!